Directory.Build.props

```diff
@@ -0,0 +1,17 @@
+<Project>
+	<PropertyGroup Condition=" '$(TargetFramework)' == 'net48' or '$(TargetFramework)' == 'netcoreapp3.1'">
+		<EnableNETAnalyzers>true</EnableNETAnalyzers>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net6.0;netcoreapp3.1;net48;</TargetFrameworks>
```

Comment:
@NeilSchiefer I just noticed, shouldn't you have `<LangVersion>latest</LangVersion>` and `<Nullable>enable</Nullable>` in your Directory.Build.props as part of the modernization?

Reply:
Thanks for catching that.